apps/trading/components/sidebar/sidebar.tsx

```diff
@@ -34,7 +34,12 @@ export const Sidebar = ({ pinnedAssets }: { pinnedAssets?: string[] }) => {
       <SidebarAccordion
         type="single"
         value={view}
-        onValueChange={(x: ViewType) => setView(x)}
+        onValueChange={(view: ViewType) => {
```

Comment:
This doesn't look correct based on the types. Should `view` be optional?

Reply:
Nope, the argument of `onValueChange` is not optional.

Type signature: `onValueChange?(value: string): void`